docs/getting-started/app-custom-theme.md

```diff
@@ -200,22 +200,22 @@ export const createCustomThemeOverrides = (
   };
 };
 
+const overrides = createCustomThemeOverrides(lightTheme);
+
+const overriddenTheme = { ...lightTheme, overrides };
```

Comment:
@Rugvip , Please let me know if this was done incorrectly; I took reference from `packages/theme/src/baseTheme.ts` on how overrides were applied.

Reply:
I think it should be like it was tbh

Reply:
Updated! For some reason I thought I couldn't do it that way, but it was a silly mistake.

I've confirmed that the overriding the theme per the latest docs works on my instance.